es-core/src/components/TextComponent.cpp

```diff
@@ -147,6 +150,9 @@ void TextComponent::render(const Transform4x4f& parentTrans)
 			case ALIGN_RIGHT:
 				Renderer::drawRect(mSize.x() - mTextCache->metrics.size.x(), 0.0f, mTextCache->metrics.size.x(), mTextCache->metrics.size.y(), 0x00000033, 0x00000033);
 				break;
+			default:
```

Comment:
Added default case to resolve compiler warning.

/home/devel/temp/EmulationStation_Stable/es-core/src/components/TextComponent.cpp:139:11: warning: en
umeration values 'ALIGN_TOP' and 'ALIGN_BOTTOM' not handled in switch [-Wswitch]
                        switch(mHorizontalAlignment)
                               ^~~~~~~~~~~~~~~~~~~~